v1/feed.yaml

```diff
@@ -71,18 +71,13 @@ paths:
                   - views: /.+/
                     rank: /.+/
                     title: /.+/
-                    pageid: /.+/
-                    normalizedtitle: /.+/
               random:
                 items:
                   - title: /.+/
               news:
                 - story: /.+/
                   links:
-                    - pageid: /.+/
-                      ns: /.+/
-                      normalizedtitle: /.+/
```

Comment:
All we really need to do is remove the pageid and ns fields from the expected response for news links.  These aren't used in the app.